powerauth-java-server/src/main/java/io/getlime/security/powerauth/app/server/database/model/entity/ActivationHistoryEntity.java

```diff
@@ -60,7 +70,13 @@
     private Date timestampCreated;
 
     /**
-     * Default constructor.
+     * Current {@link ActivationRecordEntity#getVersion()} specified whenever an activation history event is audited.
+     */
+    @Column(name = "version", nullable = false)
```

Comment:
S tím `nullable = false` si nejsem jistý, ale dávalo by mi to tak smysl, že aktivace sice nemusí mít vyplněné, ale v případě auditování už ano.

Reply:
Taky by se to slušelo vystavit na API, ale o tom issue nemluvím, tak jsem si to zatím nechal do zásoby.

Reply:
I think `nullable=false` is useful for the activation history, but we cannot use it for activation record where initially the value is expected to be `null`.

Reply:
Note that this setting will not work for existing DB rows. We either need to add steps for resolving the version for existing DB rows into migration guide or allow the `null` value. Either way works for me.

Reply:
Ouch, you are right, I tested the update script on an empty database :( Anyway, I would prefer to keep NOT NULL constraint. I am not sure, what value I can use for default.

Reply:
The activation version for existing history records can be taken from the `pa_activation` table. We'll discuss whether this makes sense or we will rather use the `null` value because clearly we cannot reconstruct information whether upgrade happened for existing records. I'll get back to you on this.

Reply:
- [ ] disable `not null` constraint or provide meaningful default

Reply:
I disabled `not null` constraint.